panther_lights/src/lights_driver_node.cpp

```diff
@@ -235,7 +236,10 @@ void DriverNode::DiagnoseLights(diagnostic_updater::DiagnosticStatusWrapper & st
   unsigned char error_level{diagnostic_updater::DiagnosticStatusWrapper::OK};
   std::string message{"Control over LEDs granted."};
 
-  if (led_control_status_ != LEDControlStatus::GRANTED) {
+  if (
+    led_control_status_ != LEDControlStatus::GRANTED &&
+    led_control_status_ != LEDControlStatus::PENDING &&
+    led_control_status_ != LEDControlStatus::IDLE) {
     error_level = diagnostic_updater::DiagnosticStatusWrapper::ERROR;
     message = "Control over LEDs not granted, driver is not functional!";
   }
```

Comment:
If `PENDING` or `IDLE` the control is not yet granted so we shouldn't `std::string message{"Control over LEDs granted."};` .